languages/csharp/exercises/concept/Interfaces/.meta/design.md

```diff
@@ -0,0 +1,36 @@
+# Design
+
+## Goal
+The goal of this exercise is to teach the student the Concept of Interfaces in C#.
+
+## Learning objectives
+
+- Know what interface is.
+- Know the characteristics of an interface.
+- Know how interface faciliate multiple inheritance.
```

Comment:
This is from Microsoft documents 
"By using interfaces, you can, for example, include behavior from multiple sources in a class. That capability is important in C# because the language doesn't support multiple inheritance of classes." [https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/]
hence thought of include it.

Reply:
Right. So this is mostly just a wording things I think. Classes don't support multiple inheritance, but do support implementing multiple interfaces. So in that sense I feel that they don't facilitate multiple inheritance, but are more like an alternative.

Reply:
I agree, but I have hear people refere interface when it's come to multiple inheritance.
What do you think ? should we remove that sentence? Please advise.

Reply:
> What do you think ? should we remove that sentence? Please advise.

I think we can mention this somewhere in the `after.md` document, but I think it can be removed from the learning goals.